ui/narrative/methods/run_Cuffdiff/display.yaml

```diff
@@ -29,6 +29,11 @@ parameters :
       Expression Set
     short-hint : |
       Select the expression set associated with transcripts assembled by Cufflinks or Stringtie
+  input_type:
+    ui-name : |
+      Data Type
```

Comment:
Isn't this more a "Feature" type rather than "Data" type - the choice is between gene or transcript, right?